src/pages/Dashboard/Notifications-Tile.vue

```diff
@@ -51,7 +51,10 @@ export default {
     },
     where() {
       let where = {
-        tenant_id: { _eq: this.tenant?.id },
+        _or: [
```

Comment:
is this an exclusive or?

Reply:
I'm wondering about the case where a user has multiple tenants and this runs
```
{ tenant_id: { _eq: null } },
{ tenant_id: { _is_null: true } 
```

Reply:
I think the query will fail in that specific case as you're expecting, since `== null && !== null` can't both be true. But a whole lot else will be failing if `tenant.id == null` so I'm not super worried about that here.

Reply:
@znicholasbrown Hasura matches _everything_ when you use `_eq: null` (hence the `_is_null` special comparison)

Reply:
This was resolved offline^